superset-frontend/src/components/ReportModal/HeaderReportActionsDropdown/index.tsx

```diff
@@ -46,21 +47,16 @@ export default function HeaderReportActionsDropDown({
   chart?: ChartState;
 }) {
   const dispatch = useDispatch();
-  const report: AlertObject = useSelector<any, AlertObject>(state => {
+
+  const findReport = useSelector<any, AlertObject>(state => {
     if (dashboardId) {
       return state.reports.dashboards?.[dashboardId];
     }
     if (chart?.id) {
-      return state.reports.charts?.[chart.id];
+      return state.reports.charts?.[chart?.id];
```

Comment:
I don't think you need the optional chaining here, since you check for `chart` on line 55.